README.md

```diff
@@ -555,6 +555,31 @@ if (s == NATS_OK)
     s = natsConnection_Connect(&nc, opts);
 ```
 
+You can sign any content and get the signature in return. The connection must have been created with the `natsOptions_SetUserCredentialsFromFiles()` option for that to work.
+```c
+    s = natsOptions_Create(&opts);
+    if (s == NATS_OK)
+        s = natsOptions_SetUserCredentialsFromFiles(opts, "user.creds", NULL);
+    if (s == NATS_OK)
+        s = natsConnection_Connect(&nc, opts);
+
+    // Sign some arbitrary content
+    const unsigned char *content   = (const unsigned char*) "hello";
+    int                 contentLen = 5;
+    unsigned char		*sig       = NULL;
+    int					sigLen     = 0;
+
+    s = natsConnection_Sign(nc, content, contentLen, &sig, &sigLen);
```

Comment:
sigLen is fixed yes, 64 bytes?

Reply:
Yes, so should I remove that from function and document that returned signature is 64 bytes?

Reply:
I think so.